itou/siae_evaluations/templatetags/siae_evaluations_siae_tags.py

```diff
@@ -44,33 +44,36 @@ def action_required_badge(content):
 TODO_BADGE = action_required_badge("À traiter")
 UPLOADED_BADGE = action_required_badge("Justificatifs téléversés")
 
-BADGES = {
-    EvaluatedJobApplicationsState.PENDING: TODO_BADGE,
-    EvaluatedJobApplicationsState.PROCESSING: info_badge("En cours"),
-    EvaluatedJobApplicationsState.SUBMITTED: SUBMITTED_BADGE,
-    EvaluatedJobApplicationsState.UPLOADED: UPLOADED_BADGE,
-    # When the institution evaluation is in progress, the institution updates
-    # the job application state. Don’t reveal the new state before the review
-    # is published.
-    EvaluatedJobApplicationsState.ACCEPTED: SUBMITTED_BADGE,
-    EvaluatedJobApplicationsState.REFUSED: SUBMITTED_BADGE,
-}
-
-REVIEWED_BADGES = {
-    EvaluatedJobApplicationsState.ACCEPTED: ACCEPTED_BADGE,
-    EvaluatedJobApplicationsState.UPLOADED: UPLOADED_BADGE,
-    EvaluatedJobApplicationsState.SUBMITTED: SUBMITTED_BADGE,
-    EvaluatedJobApplicationsState.REFUSED: REFUSED_BADGE,
-    EvaluatedJobApplicationsState.REFUSED_2: REFUSED_BADGE,
-    EvaluatedJobApplicationsState.PROCESSING: TODO_BADGE,
-    EvaluatedJobApplicationsState.PENDING: PENDING_AFTER_REVIEW_BADGE,
-}
+
+def get_employer_badges(amicable_stage):
```

Comment:
On a déjà pas mal d'`adversarial_stage` dans la base de code et 0 `amicable_stage`: cela me semble plus clair de rester sur le nom connu (avec `adversarial_stage = not amicable_stage`).